cookbooks/aws-parallelcluster-platform/test/controls/intel_hpc_spec.rb

```diff
@@ -24,7 +24,7 @@
 end
 
 control 'tag:config_intel_hpc_enough_space_on_root_volume' do
-  only_if { !instance.custom_ami? }
+  only_if { !instance.custom_ami? && (os_properties.centos7? && os_properties.x86?) }
```

Comment:
This is also an issue. We added this but it’s incorrect. It should be 
only_if { !instance.custom_ami? && !os_properties.on_docker